linkup/client.py

```diff
@@ -268,11 +269,18 @@ def _raise_linkup_error(self, response: httpx.Response) -> None:
                     f"Original error message: {error_msg}."
                 )
             elif response.status_code == 429:
-                raise LinkupInsufficientCreditError(
-                    "The Linkup API returned an insufficient credit error (429). Make sure "
-                    "you haven't exhausted your credits.\n"
-                    f"Original error message: {error_msg}."
-                )
+                if code == "INSUFFICIENT_FUNDS_CREDITS":
+                    raise LinkupInsufficientCreditError(
+                        "The Linkup API returned an insufficient credit error (429). Make sure "
+                        "you haven't exhausted your credits.\n"
+                        f"Original error message: {error_msg}."
+                    )
+                if code == "TOO_MANY_REQUESTS":
```

Comment:
```suggestion
                elif code == "TOO_MANY_REQUESTS":
```
(otherwise it seems to suggest that both conditions could happen whereas they are not compatible)